dashboard.php

```diff
@@ -17,8 +17,52 @@
             </ul>
         </nav>
     </header>
+    <?php 
+    require_once('php/connection.php');
+    session_start();
+    $dsn = DB_DSN;
+    $user = DB_USERNAME;
+    $pass = DB_PASSWORD;
+
+    $opt = [
+        PDO::ATTR_ERRMODE            => PDO::ERRMODE_EXCEPTION,
+        PDO::ATTR_DEFAULT_FETCH_MODE => PDO::FETCH_ASSOC,
+        PDO::ATTR_EMULATE_PREPARES   => false,
+    ];
+    $pdo = new PDO($dsn, $user, $pass, $opt);
+
+    $userID = $_SESSION["userID"];
+    $conn = newConn();
+
+    //Get the logged in user's details, things like the first name and studentID
+    $stmt = $conn->prepare("CALL GetStudentData(:userID)");
+    $stmt->bindValue(":userID", $userID, PDO::PARAM_STR);
+    $stmt->execute();
+    $studentData = $stmt->fetch(PDO::FETCH_ASSOC);
+
+    $userName = $studentData['firstname'];
+    $studentID = $studentData['studentID'];
+
+   //Query to get the tests
+   $sql = "SELECT quizzes.quizID, quizzes.title, subjects.name, studentQuizLink.TIMESTAMP, studentQuizLink.correctCount, studentQuizLink.questionCount, studentQuizLink.points, studentQuizLink.completed
+   FROM quizzes 
+   JOIN subjects ON quizzes.subjectID = subjects.subjectID
+   JOIN studentQuizLink ON quizzes.quizID = studentQuizLink.quizID AND studentQuizLink.studentID = :studentID";
```

Comment:
Don't include SQL in the code, this should be within a prepared function/procedure within PHPMyAdmin

Reply:
What procedure would I use? I tried "GetAssignedQuiz" and couldn't get it to work

Reply:
Ok thanks. I'll add that to my code now

Reply:
Done. It now also returns TIMESTAMP AS 'dateSet'.
I've also ordered it by the TIMESTAMP, so the most recently set tests will be at the top of the list.